### PR TITLE
Add post about Web Retrieval

### PR DESCRIPTION
**Pull request by ZanSara**
The thumbnail is from Wikipedia, copyright free. Let me know if and how I should provide attribution.

### CODE REVIEW

File: content/blog/the-world-of-web-rag/index.md [hunk 1]
```diff
@@ -0,0 +1,359 @@
...
+In this post, we will see how to build a Web RAG application: a RAG pipeline that can search the Web for the information needed to answer your questions.
+
+> 💡 *Do you want to see the code in action? Check out the [Colab notebook](https://colab.research.google.com/drive/1dGMPxReo730j7_zQDZOu-0SGf-pk4XDL?usp=sharing) or the [gist](https://gist.github.com/ZanSara/0907a8f3ae19f62998cc061ed6e8ce53).*
+
+> <i>⚠️ **Warning:**</i> *This code was tested on `haystack-ai==0.130.0`. Haystack 2.0 is still unstable, so later versions might introduce breaking changes without notice until Haystack 2.0 is officially released. The concepts and components, however, stay the same.*
+
+
+# Searching the Web
+
+As we've seen [earlier](#), a Haystack RAG Pipeline is made of three components: a Retriever, a PromptBuilder, and a Generator, and looks like this:
```
Comment (ZanSara) [inline content/blog/the-world-of-web-rag/index.md:27]: This was linking to a post that we haven't published here yet. LMK what to point it to.

File: content/blog/the-world-of-web-rag/index.md [hunk 2]
```diff
@@ -0,0 +1,359 @@
...
+# }
+```
+
+First, let's notice that `LinkContentFetcher` outputs a list of `ByteStream` objects. `ByteStream` is a Haystack abstraction that makes handling binary streams and files equally easy. When a component produces `ByteStream` as output, you can directly pass these objects to a Converter component that can extract its textual content without saving such binary content to a file.
+
+These features come in handy to connect `LinkContentFetcher` to a component we've already met before: `HTMLToDocument`.
+
+# Processing the page
+
+In a [previous post](#), we've seen how Haystack can convert web pages into clean Documents ready to be stored in a Document Store. We will reuse many of the components we have discussed there, so if you missed it, make sure to check it out.
```
Comment (ZanSara) [inline content/blog/the-world-of-web-rag/index.md:139]: This was linking to a post that we haven't published here yet. LMK what to point it to.

File: content/blog/the-world-of-web-rag/index.md [hunk 3]
```diff
@@ -0,0 +1,359 @@
...
+alt_image: A person water-skiing in front of the Rose Island sea platform in a vintage picture.
+toc: True
+date: 2023-11-09
+last_updated:  2023-11-09
+authors:
+  - Sara Zanzottera
+tags: ["Haystack 2.0"]
+---	
+
+In an earlier post of the Haystack 2.0 series, we've seen how to build RAG and indexing pipelines. An application that uses these two pipelines is practical if you have an extensive, private collection of documents and need to perform RAG on such data only. However, in many cases, you may want to get data from the Internet: from news outlets, documentation pages, and so on.
```
Comment (TuanaCelik) [inline content/blog/the-world-of-web-rag/index.md:16]: I would rephrase because we will probably post this before the others: "An application that uses these two pipelines is practical if you have an extensive, private collection of documents and need to perform RAG on such data only. One pipeline to index documents to your own, private document store (dubbed an indexing pipeline), and another to query it with a language model (the querying pipeline). However,..."

File: content/blog/the-world-of-web-rag/index.md (hunk 1 above)
Comment (TuanaCelik) [reply content/blog/the-world-of-web-rag/index.md:27]: just remove the first bit omi, Start with 'A Haystack RAG pipeline...'

File: content/blog/the-world-of-web-rag/index.md (hunk 2 above)
Comment (TuanaCelik) [reply content/blog/the-world-of-web-rag/index.md:139]: Again, same as previous comment, we will probably publish this first so we can remove this line?

**Review by TuanaCelik — CHANGES_REQUESTED** [review]
I've added some comments, @bilgeyucel might have more. Also, do we have to change the headers to h2? Not sure.
